Languages/es.xml

```diff
@@ -755,50 +756,50 @@ Gracias a: Oleada, Jhovany200, nomemolesteis@gmail.com por la traducción.
 
     <String Target="隐藏可执行文件小盾牌" Link="Ocultar el UAC"/>
 
-    <String Target="创建快捷方式时不添&quot;快捷方式&quot;文字（by 518516.net）" Link="Prevenir añadir texto en &quot;Acceso directo&quot; al crear un acceso directo (by 518516.net)"/>
+    <String Target="创建快捷方式时不添&quot;快捷方式&quot;文字（by 518516.net）" Link="Prevenir añadir texto en &quot;Acceso directo&quot; al crear un acceso directo (por 518516.net)"/>
 
-    <String Target="隐藏此电脑中视频、图片、文档、下载、音乐、桌面六个文件夹" Link="Ocultar carpetas: Escritorio, Videos, Imágenes, Documentos, Descargas, Imágenes y Música en este PC"/>
+    <String Target="隐藏此电脑中视频、图片、文档、下载、音乐、桌面六个文件夹" Link="Ocultar carpetas: Escritorio, Videos, Imágenes, Documentos, Descargas, etc."/>
```

Comment:
Why are they deleted? 

"Ocultar carpetas: Escritorio, Videos, Imágenes, Documentos, Descargas, Imágenes y Música en este PC" is better.

Do you think so?

Reply:
the majority of people know what follows, "etc" is in Spanish means to continue.......
since it does not enter everything.